docs/rules/require-post-message-target-origin.md

```diff
@@ -0,0 +1,19 @@
+# Enforce using the targetOrigin argument with `Window.postMessage()`
```

Comment:
```suggestion
# Enforce using the `targetOrigin` argument with `window.postMessage()`
```

Reply:
MDN wrote `Window`, I wonder where is the `postMessage` method, it's not in `Window.prototype`